wordpress-mobile/WordPress-iOS
# Update credentials to include Tenor API key

Ref https://github.com/wordpress-mobile/WordPress-iOS/issues/13803

This updates credentials files to include the Tenor API key from the secrets repo.

To test:

Verify the key is generated:
- In the project directory, go to `Derived Sources`. In `ApiCredential.m`, verify there is an entry for `tenorApiKey`.

Verify GIF searching:
- Go to Media > Add > Free GIF Library.
- Enter a search term.
- Verify search results are displayed.

![Simulator Screen Shot - iPhone 11 Pro Max - 2020-06-11 at 10 23 31](https://user-images.githubusercontent.com/1816888/84413337-b2861b00-abcd-11ea-8cb7-a631a6985788.png)


PR submission checklist:
